Library/Homebrew/cmd/test-bot.rb

```diff
@@ -583,7 +583,7 @@ def formula(formula_name)
           end
         end
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
-        testable_dependents.each do |dependent|
+        dependents.each do |dependent|
```

Comment:
Good point. Yeah, testing linkage on non-bottled formulae is probably pointless. I'll modify it to do only bottled ones.